platform_channels/android/app/src/main/kotlin/dev/flutter/platform_channels/MainActivity.kt

```diff
@@ -40,5 +43,15 @@ class MainActivity : FlutterActivity() {
         val accelerometerSensor: Sensor = sensorManger.getDefaultSensor(Sensor.TYPE_ACCELEROMETER)
         EventChannel(flutterEngine.dartExecutor, "eventChannelDemo")
                 .setStreamHandler(AccelerometerStreamHandler(sensorManger, accelerometerSensor))
+
+        // Registers a MessageHandler for BasicMessageChannel to receive a message from dart side and send
+        // the image data in reply.
```

Comment:
nit:

        // Registers a MessageHandler for BasicMessageChannel to receive
        // a message from Dart and send image data in reply.